cmd/mayactl/app/command/version_test.go

```diff
@@ -0,0 +1,46 @@
+package command
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/openebs/maya/pkg/version"
+
+	utiltesting "k8s.io/client-go/util/testing"
+)
+
+var (
+	response2 = `[{"tag_name": "0.5.4"}]`
+)
+
+func TestCheckLatestVersion(t *testing.T) {
+	cases := []struct {
+		installed string
```

Comment:
can you use map[string]struct instead of []struct, the benefit of using map is you can write your test case name and it will increase the code readability too.